packages/slotted-label-mixin/slotted-label-mixin.ts

```diff
@@ -0,0 +1,28 @@
+import { LitElement, PropertyValues } from 'lit-element';
+import { Constructor } from '@vaadin/mixin-utils';
+
+const isEmpty = (nodes: Node[]) =>
+  nodes.every(node => node.nodeType === Node.TEXT_NODE && !node.textContent?.trim());
+
+export const SlottedLabelMixin = <T extends Constructor<LitElement>>(base: T): T => {
+  class SlottedLabel extends base {
+    protected firstUpdated(props: PropertyValues) {
+      super.firstUpdated(props);
+
+      const label = this.renderRoot.querySelector('[part~="label"]');
```

Comment:
I guess this variable is no longer needed? You could query the slot element directly with `this.renderRoot.querySelector('[part~="label"] slot');`

Reply:
Thanks, done.